app/javascript/components/lease_ups/LeaseUpApplicationsPage.js

```diff
@@ -89,11 +97,52 @@ const LeaseUpApplicationsPage = () => {
     subStatus: null
   })
 
-  const [{ breadcrumbData }, actions] = useContext(AppContext)
   const isMountedRef = useIsMountedRef()
 
-  // grab the listing id from the url: /lease-ups/listings/:listingId
-  const { listingId } = useParams()
+  const [{ reportId, listingPreferences }, setListingState] = useStateObject({})
+  useAsyncOnMount(() => getListing(listingId), {
+    onSuccess: (listing) => {
+      setListingState({
+        reportId: listing.report_id,
+        listingPreferences: getPreferences(listing)
+      })
+
+      actions.applicationsPageLoadComplete(listing)
+    }
+  })
+
+  useAsync(
```

Comment:
This is basically the same as loadPage, except it's triggered by state change instead of a reactTable callback.